### PR TITLE
system-variables: update tidb_enable_paging, tidb_min_paging_size, tidb_max_paging_size

### PR DESCRIPTION
**Pull request by tiancaiamao**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by tiancaiamao** [comment]
PTAL @XuHuaiyu

**Review by XuHuaiyu — APPROVED** [review]

File: system-variables.md [hunk 1]
```diff
@@ -2156,7 +2155,11 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 默认值：`128`
 - 范围：`[1, 9223372036854775807]`
 - 单位：行
-- 这个变量用来设置 coprocessor 协议中 paging size 的最小的行数。请合理设置该值，设置过小，TiDB 与 TiKV 的 RPC 交互会更频繁；设置过大，IndexLookup 带 Limit 场景会出现性能下降。
+- 这个变量用来设置 coprocessor 协议中 paging size 的最小的行数。请合理设置该值，设置过小，TiDB 与 TiKV 的 RPC 交互会更频繁；设置过大，IndexLookup 带 Limit 场景会出现性能下降。默认值对于 TP 场景较友好，如果业务是使用 TiKV 存储引擎，执行偏 AP 负载时，可以考虑将变量值调大，有可能获得更好的性能。
```
Comment (qiancai) [inline system-variables.md:2158]:
```suggestion
- 这个变量用来设置 coprocessor 协议中 paging size 的最小的行数。请合理设置该值，设置过小，TiDB 与 TiKV 的 RPC 交互会更频繁；设置过大，IndexLookup 带 Limit 场景会出现性能下降。该变量的默认值对于 OLTP 场景较友好，如果业务只使用了 TiKV 作为存储引擎，当执行偏 OLAP 的负载时，可以考虑将变量值调大，有可能获得更好的性能。
```

File: system-variables.md [hunk 2]
```diff
@@ -2156,7 +2155,11 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
+
+![](https://user-images.githubusercontent.com/1420062/201061302-d88f4702-cb10-4523-8500-02389769d774.png)
+
+`tidb_enable_paging`, `tidb_min_paging_size`, `tidb_max_paging_size` 等相关变量对 TPCH 的性能影响，纵轴是执行时间，越小越好。
```
Comment (qiancai) [inline system-variables.md:2162]:
```suggestion
[`tidb_enable_paging`](/system-variables.md#tidb_max_paging_size-从-v630-版本开始引入), `tidb_min_paging_size`, [`tidb_max_paging_size`](/system-variables.md#tidb_max_paging_size-从-v630-版本开始引入) 等相关变量对 TPCH 的性能影响如上图所示，纵轴是执行时间，越小越好。
```

File: system-variables.md [hunk 3]
```diff
@@ -2156,7 +2155,11 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 默认值：`128`
 - 范围：`[1, 9223372036854775807]`
 - 单位：行
-- 这个变量用来设置 coprocessor 协议中 paging size 的最小的行数。请合理设置该值，设置过小，TiDB 与 TiKV 的 RPC 交互会更频繁；设置过大，IndexLookup 带 Limit 场景会出现性能下降。
+- 这个变量用来设置 coprocessor 协议中 paging size 的最小的行数。请合理设置该值，设置过小，TiDB 与 TiKV 的 RPC 交互会更频繁；设置过大，IndexLookup 带 Limit 场景会出现性能下降。默认值对于 TP 场景较友好，如果业务是使用 TiKV 存储引擎，执行偏 AP 负载时，可以考虑将变量值调大，有可能获得更好的性能。
+
+![](https://user-images.githubusercontent.com/1420062/201061302-d88f4702-cb10-4523-8500-02389769d774.png)
```
Comment (qiancai) [inline system-variables.md:2160]: 这个图打开只能看到上半部分，是否可以下载下来，加到文档 /meida 文件夹下呢？

**Comment by qiancai** [comment]
@tiancaiamao I've added some minor comments. PTAL. Thanks.

**Comment by qiancai** [comment]
@tiancaiamao I made some minor changes from the users' perspective in https://github.com/pingcap/docs-cn/pull/11970/commits/29abdee945fbba8f69c2aa842ae8d7fce4aa2a57. Would you please help take a look at it if the changes are fine? Thanks.

File: system-variables.md [hunk 4]
```diff
@@ -2155,11 +2160,11 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 默认值：`128`
 - 范围：`[1, 9223372036854775807]`
 - 单位：行
-- 这个变量用来设置 coprocessor 协议中 paging size 的最小的行数。请合理设置该值，设置过小，TiDB 与 TiKV 的 RPC 交互会更频繁；设置过大，IndexLookup 带 Limit 场景会出现性能下降。默认值对于 TP 场景较友好，如果业务是使用 TiKV 存储引擎，执行偏 AP 负载时，可以考虑将变量值调大，有可能获得更好的性能。
+- 这个变量用来设置 coprocessor 协议中 paging size 的最小的行数。请合理设置该值，设置过小，TiDB 与 TiKV 的 RPC 交互会更频繁；设置过大，IndexLookup 带 Limit 场景会出现性能下降。该变量的默认值对于 OLTP 场景较友好，如果业务只使用了 TiKV 作为存储引擎，当执行偏 OLAP 的负载时，可以考虑将变量值调大，有可能获得更好的性能。
 
 ![](https://user-images.githubusercontent.com/1420062/201061302-d88f4702-cb10-4523-8500-02389769d774.png)
 
-`tidb_enable_paging`, `tidb_min_paging_size`, `tidb_max_paging_size` 等相关变量对 TPCH 的性能影响，纵轴是执行时间，越小越好。
+[`tidb_enable_paging`](/system-variables.md#tidb_max_paging_size-从-v630-版本开始引入), `tidb_min_paging_size`, [`tidb_max_paging_size`](/system-variables.md#tidb_max_paging_size-从-v630-版本开始引入) 等相关变量对 TPCH 的性能影响如上图所示，纵轴是执行时间，越小越好。
```
Comment (XuHuaiyu) [inline system-variables.md]: tidb_min_paging_size 需要加上 link 吗？

Comment (qiancai) [reply system-variables.md]: 谢谢，应该是不需要的，这就是在它下面的描述，加了就是自己链接自己了

**Review by XuHuaiyu — APPROVED** [review]

File: system-variables.md [hunk 5]
```diff
@@ -1406,10 +1406,13 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
-- 默认值：`OFF`
-- 这个变量用于控制 `IndexLookUp` 算子是否使用分页 (paging) 方式发送 Coprocessor 请求。
-- 适用场景：对于使用 `IndexLookUp` 和 `Limit` 并且 `Limit` 无法下推到 `IndexScan` 上的读请求，可能会出现读请求的延迟高、TiKV 的 Unified read pool CPU 使用率高的情况。在这种情况下，由于 `Limit` 算子只需要少部分数据，开启 `tidb_enable_paging`，能够减少处理数据的数量，从而降低延迟、减少资源消耗。
-- 开启 `tidb_enable_paging` 后，`Limit` 无法下推且数量小于 `960` 的 `IndexLookUp` 请求会使用 paging 方式发送 Coprocessor 请求。`Limit` 的值越小，优化效果会越明显。
+- 默认值：`ON`
+- 这个变量用于控制是否使用分页 (paging) 方式发送 Coprocessor 请求。对于 [v5.4.0, v6.2.0) 区间的 TiDB 版本，该变量只对 `IndexLookup` 算子生效；对于 v6.2.0 以及之后的版本，该变量对全局生效。从 v6.4.0 版本开始，该变量默认值由 `OFF` 改成 `ON`。
+- 适用场景：所有偏 OLTP 的场景推荐开启。对于使用 `IndexLookUp` 和 `Limit` 并且 `Limit` 无法下推到 `IndexScan` 上的读请求，可能会出现读请求的延迟高、TiKV 的 Unified read pool CPU 使用率高的情况。在这种情况下，由于 `Limit` 算子只需要少部分数据，开启 `tidb_enable_paging`，能够减少处理数据的数量，从而降低延迟、减少资源消耗。此外，对于 Dumpling 数据导出或者全表扫这类的场景，开启 paging 后可以有效降低 TiDB 进程的内存消耗。
```
Comment (Oreoxmt) [inline system-variables.md:1411]:
```suggestion
- 适用场景：

    - 推荐在所有偏 OLTP 的场景下使用 paging。
    - 对于使用 `IndexLookUp` 和 `Limit` 并且 `Limit` 无法下推到 `IndexScan` 上的读请求，可能会出现读请求的延迟高、TiKV 的 Unified read pool CPU 使用率高的情况。在这种情况下，由于 `Limit` 算子只需要少部分数据，开启 `tidb_enable_paging` 能够减少处理数据的数量，从而降低延迟、减少资源消耗。
    - 对于 Dumpling 数据导出或者全表扫描这类的场景，开启 paging 后可以有效降低 TiDB 进程的内存消耗。
```

File: system-variables.md [hunk 6]
```diff
@@ -2156,7 +2159,11 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 默认值：`128`
 - 范围：`[1, 9223372036854775807]`
 - 单位：行
-- 这个变量用来设置 coprocessor 协议中 paging size 的最小的行数。请合理设置该值，设置过小，TiDB 与 TiKV 的 RPC 交互会更频繁；设置过大，IndexLookup 带 Limit 场景会出现性能下降。
+- 这个变量用来设置 coprocessor 协议中 paging size 的最小的行数。请合理设置该值，设置过小，TiDB 与 TiKV 的 RPC 交互会更频繁；设置过大，IndexLookup 带 Limit 场景会出现性能下降。该变量的默认值对于 OLTP 场景较友好，如果业务只使用了 TiKV 作为存储引擎，当执行偏 OLAP 的负载时，可以考虑将变量值调大，有可能获得更好的性能。
+
+![Paging size impact on TPCH](/media/paging-size-impact-on-tpch.png)
+
+开启 [`tidb_enable_paging`](/system-variables.md#tidb_max_paging_size-从-v630-版本开始引入)时， `tidb_min_paging_size` 和 [`tidb_max_paging_size`](/system-variables.md#tidb_max_paging_size-从-v630-版本开始引入) 对 TPCH 的性能影响如上图所示，纵轴是执行时间，越小越好。
```
Comment (Oreoxmt) [inline system-variables.md:2166]:
```suggestion
开启 [`tidb_enable_paging`](/system-variables.md#tidb_max_paging_size-从-v630-版本开始引入)时，`tidb_min_paging_size` 和 [`tidb_max_paging_size`](/system-variables.md#tidb_max_paging_size-从-v630-版本开始引入) 对 TPCH 的性能影响如上图所示，纵轴是执行时间，越小越好。
```

File: system-variables.md [hunk 7]
```diff
@@ -1406,10 +1406,17 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
-- 这个变量用于控制 `IndexLookUp` 算子是否使用分页 (paging) 方式发送 Coprocessor 请求。
-- 适用场景：对于使用 `IndexLookUp` 和 `Limit` 并且 `Limit` 无法下推到 `IndexScan` 上的读请求，可能会出现读请求的延迟高、TiKV 的 Unified read pool CPU 使用率高的情况。在这种情况下，由于 `Limit` 算子只需要少部分数据，开启 `tidb_enable_paging`，能够减少处理数据的数量，从而降低延迟、减少资源消耗。
-- 开启 `tidb_enable_paging` 后，`Limit` 无法下推且数量小于 `960` 的 `IndexLookUp` 请求会使用 paging 方式发送 Coprocessor 请求。`Limit` 的值越小，优化效果会越明显。
+- 默认值：`ON`
+- 这个变量用于控制是否使用分页 (paging) 方式发送 Coprocessor 请求。对于 [v5.4.0, v6.2.0) 区间的 TiDB 版本，该变量只对 `IndexLookup` 算子生效；对于 v6.2.0 以及之后的版本，该变量对全局生效。从 v6.4.0 版本开始，该变量默认值由 `OFF` 改成 `ON`。
+- 适用场景：
+
+    - 推荐在所有偏 OLTP 的场景下使用 paging。
+    - 对于使用 `IndexLookUp` 和 `Limit` 并且 `Limit` 无法下推到 `IndexScan` 上的读请求，可能会出现读请求的延迟高、TiKV 的 Unified read pool CPU 使用率高的情况。在这种情况下，由于 `Limit` 算子只需要少部分数据，开启 `tidb_enable_paging` 能够减少处理数据的数量，从而降低延迟、减少资源消耗。
+    - 对于 Dumpling 数据导出或者全表扫描这类的场景，开启 paging 后可以有效降低 TiDB 进程的内存消耗。
```
Comment (qiancai) [inline system-variables.md:1415]:
```suggestion
    - 对于 [Dumpling](/dumpling-overview.md) 数据导出或者全表扫描这类的场景，开启 paging 后可以有效降低 TiDB 进程的内存消耗。
```

File: system-variables.md [hunk 8]
```diff
@@ -1406,10 +1406,17 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
-- 默认值：`OFF`
-- 这个变量用于控制 `IndexLookUp` 算子是否使用分页 (paging) 方式发送 Coprocessor 请求。
-- 适用场景：对于使用 `IndexLookUp` 和 `Limit` 并且 `Limit` 无法下推到 `IndexScan` 上的读请求，可能会出现读请求的延迟高、TiKV 的 Unified read pool CPU 使用率高的情况。在这种情况下，由于 `Limit` 算子只需要少部分数据，开启 `tidb_enable_paging`，能够减少处理数据的数量，从而降低延迟、减少资源消耗。
-- 开启 `tidb_enable_paging` 后，`Limit` 无法下推且数量小于 `960` 的 `IndexLookUp` 请求会使用 paging 方式发送 Coprocessor 请求。`Limit` 的值越小，优化效果会越明显。
+- 默认值：`ON`
+- 这个变量用于控制是否使用分页 (paging) 方式发送 Coprocessor 请求。对于 [v5.4.0, v6.2.0) 区间的 TiDB 版本，该变量只对 `IndexLookup` 算子生效；对于 v6.2.0 以及之后的版本，该变量对全局生效。从 v6.4.0 版本开始，该变量默认值由 `OFF` 改成 `ON`。
+- 适用场景：
+
+    - 推荐在所有偏 OLTP 的场景下使用 paging。
+    - 对于使用 `IndexLookUp` 和 `Limit` 并且 `Limit` 无法下推到 `IndexScan` 上的读请求，可能会出现读请求的延迟高、TiKV 的 Unified read pool CPU 使用率高的情况。在这种情况下，由于 `Limit` 算子只需要少部分数据，开启 `tidb_enable_paging` 能够减少处理数据的数量，从而降低延迟、减少资源消耗。
```
Comment (Oreoxmt) [inline system-variables.md:1414]:
```suggestion
    - 对于使用 `IndexLookUp` 和 `Limit` 并且 `Limit` 无法下推到 `IndexScan` 上的读请求，可能会出现读请求的延迟高、TiKV 的 Unified Read Pool CPU 使用率高的情况。在这种情况下，由于 `Limit` 算子只需要少部分数据，开启 `tidb_enable_paging` 能够减少处理数据的数量，从而降低延迟、减少资源消耗。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Review by qiancai — APPROVED** [review]

**Pull request merged**